content/projects/metabake.md

```diff
@@ -0,0 +1,46 @@
+---
+title: MetaBake.org
+repo: metabake/_mBake
+homepage: https://metabake.org
+language:
+  - Pug
+license:
+  - GNU GPL v3.0
+templates:
+  - Pug (w/ RIOTjs)
```

Comment:
Please change to "Pug".

Reply:
I'll have to do a new PR.

Reply:
You can just push an additional commit with "Pug (w/ RIOTjs)" changed to "Pug".

Reply:
I made those changes - still no joy. 
"5:51:04 PM:   SyntaxError: Unexpected end of JSON input
5:51:04 PM:   - JSON.parse
5:51:05 PM:   - fetch-archive.js:87 _callee5$"

I'd love to get on static gen home page.